### PR TITLE
Splitting BoardPage component into simpler/smaller components

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]
Nice!  :+1:

File: webapp/src/pages/boardPage/backwardCompatibilityQueryParamsRedirect.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    //         UserSettings.setLastViewId(boardId, viewId)
+    //     }
+    //
+    //     dispatch(setCurrentBoard(boardId || ''))
+    //     dispatch(setCurrentView(viewId || ''))
+    // }, [match.params.boardId, match.params.viewId, boardViews])
+    return null
+}
+
+export default BackwardCompatibilityQueryParamsRedirect
```
Comment (sbishel) [inline webapp/src/pages/boardPage/backwardCompatibilityQueryParamsRedirect.tsx:75]: Since this is commented out, can this be deleted?

**Review by sbishel — APPROVED** [review]
Just wondering if file is necessary, but otherwise looks good.

File: webapp/src/pages/boardPage/backwardCompatibilityQueryParamsRedirect.tsx (hunk 1 above)
Comment (jespino) [reply webapp/src/pages/boardPage/backwardCompatibilityQueryParamsRedirect.tsx:75]: The thing is, I don't know if we want to maintain the backward compatibility from that part, that would be easy (for older links) @chenilim do you have any preference? Preserve backward compatiblity with links that use the ?cardId=whatever, and things like that?

**Pull request merged**